app/views/admin/organizations/_form.html.haml

```diff
@@ -1,7 +1,14 @@
 = semantic_form_for(@organization, url: (@organization.new_record? ? admin_organizations_path : admin_organization_path(@organization))) do |f|
   = f.inputs name: 'Organization details' do
     = f.input :name, as: :string, required: true
-    = f.input :description, as: :text, input_html: { rows: 10 }, placeholder: 'Decribe about your organization..'
+    = f.input :description, as: :text,
+      input_html: {rows: 10, data: { provide: 'markdown-editable' } },
```

Comment:
space missed before `rows`:

`input_html: { rows: 10, data: { provide: 'markdown-editable' } },`

why are our linters not complaining? :sob: